### ISSUE TITLE
provide configuration environment

### ISSUE THREAD

**Issue by zxj11zxj11zxj11** [wilkinghoff/few-shot-open-set-eusipco2023#1]
Dear author, read your work feel benefited, could you please provide configuration environment and python edition?Thank you very much.

**Comment by wilkinghoff** [comment]
Dear zxj11zxj11zxj11, thank you for your interest! I used Python 3.6.5 and uploaded a requirement.txt file containing the packages.

**Comment by zxj11zxj11zxj11** [comment]
Thanks for your prompt reply.But there is still a question that Scikit-learn 1.3.0(https://github.com/scikit-learn/scikit-learn/releases/tag/1.3.0) in requirements.txt only seems to support python versions 3.8 to 3.11.I can't install it.Looking forward to your reply.
![image](https://github.com/wilkinghoff/few-shot-open-set-eusipco2023/assets/127805303/9ade010c-b491-4c30-92c8-8cd540296d13)

**Comment by wilkinghoff** [comment]
You are absolutely correct! My sincere apologies. I used pipreqs to generate the requirements file and the extracted scikit-learn version is wrong for some unknown reason. The correct version is 0.24.2. I also manually updated the requirements file. Please let me know if there are any other issues, thanks!

**Comment by zxj11zxj11zxj11** [comment]
Thanks for your help!And I can't find this article‘On Using Pre-Trained Embeddings for Detecting Anomalous Sounds with Limited Training Data’ as a reference. Could you please send me the website？

**Comment by wilkinghoff** [comment]
The paper is not yet publicly available. However, it should not take long until the paper is available as EUSIPCO, which is the conference the paper has been accepted to, will be held next week. If you want, I can also send you a preprint of the paper before that by mail. In this case, please contact me at: kevin.wilkinghoff@fkie.fraunhofer.de
